shared/oval/disable_interactive_boot.xml

```diff
@@ -1,34 +1,74 @@
 <def-group>
-  <definition class="compliance" id="disable_interactive_boot" version="2">
+  <definition class="compliance" id="disable_interactive_boot" version="3">
     <metadata>
-      <title>Disable Interactive Boot</title>
+      <title>Verify that Interactive Boot is Disabled</title>
       <affected family="unix">
         <platform>Red Hat Enterprise Linux 7</platform>
+        <platform>multi_platform_fedora</platform>
       </affected>
       <description>The ability for users to perform interactive startups should
       be disabled.</description>
-      <reference source="JL" ref_id="20140407" ref_url="test_attestation" />
+      <reference source="JL" ref_id="RHEL7_20160613" ref_url="test_attestation" />
+      <reference source="JL" ref_id="FEDORA23_20160613" ref_url="test_attestation" />
     </metadata>
-    <criteria>
-      <criterion test_ref="test_disable_interactive_boot" />
-      <!-- This needs to be enhanced yet to properly cover case of 'systemd.confirm_spawn=1'
-           use case on RHEL-7 and Fedora per:
-             https://rhsummit.files.wordpress.com/2014/04/summit_demystifying_systemd1.pdf Page #20
-           SSG Upstream ticket to come yet -->
+    <criteria operator="OR">
```

Comment:
@redhatrises 
This seems to be best explanation of the slight difference I was able to find: http://askubuntu.com/questions/575651/what-is-the-difference-between-grub-cmdline-linux-and-grub-cmdline-linux-default/780545#780545

As I understand it, from https://www.gnu.org/software/grub/manual/grub.html being:
* if GRUB_DISABLE_RECOVERY=true, the default kernel menu entry will be equipped also with options / settings from GRUB_CMDLINE_LINUX_DEFAULT (they would be appended after values / settings from GRUB_CMDLINE_LINUX)
* if GRUB_DISABLE_RECOVERY=false, just GRUB_CMDLINE_LINUX settings would be effective.

So IMHO we should:
* if GRUB_DISABLE_RECOVERY=false inspect just GRUB_CMDLINE_LINUX settings,
* if GRUB_DISABLE_RECOVERY=true inspect both GRUB_CMDLINE_LINUX and GRUB_CMDLINE_LINUX_DEFAULT settings (will recheck the OVAL if it's working as described here actually).